.github/workflows/tests.yml

```diff
@@ -0,0 +1,51 @@
+name: Tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: true
+      matrix:
+        php: [7.2, 7.3, 7.4, 8.0]
+        stability: [prefer-lowest, prefer-stable]
```

Comment:
I've added a `prefer-lowest` to the matrix build. This uncovered some things like the lowest `symfony/yaml` version not being compatible with PHP 8 anymore.